tests/pyret/tests/test-parse-errors.arr

```diff
@@ -74,7 +74,36 @@ end
 bad-number = [list:
   ".0",
   "~.0",
-  "foo.0"
+  "foo.0",
+  #"2.",
```

Comment:
I see.  (This is why I'd recommended these be done as JS lexer tests, rather than Pyret test cases... It'd be more obvious to see which ones don't lex, and which ones lexed as two tokens.)